src/shared_modules/dbsync/src/sqlite/sqlite_wrapper.cpp

```diff
@@ -230,6 +230,12 @@ void Statement::bind(const int32_t index, const int32_t value)
     checkSqliteResult(result, sqlite3_errmsg(m_connection->db().get()));
     ++m_bindParametersIndex;
 }
+void Statement::bind(const int32_t index, const uint32_t value)
```

Comment:
Same as previous comment, since the fix is on the manager side and we see that it only affects the MTU, I would remove this change to avoid edge conditions.